crates/bevy_ecs/src/world/mod.rs

```diff
@@ -34,11 +36,47 @@ impl Default for WorldId {
     }
 }
 
-/// [World] stores and exposes operations on [entities](Entity), [components](Component),
+/// Stores and exposes operations on [entities](Entity), [components](Component),
 /// and their associated metadata.
 /// Each [Entity] has a set of components. Each component can have up to one instance of each
 /// component type. Entity components can be created, updated, removed, and queried using a given
 /// [World].
+///
+/// # Resources
+///
+/// The world can store information not associated to entity data, called *resources*. They
```

Comment:
I prefer a slight rewording of this: "Worlds can also store Resources, which are unique instances of a given type that don't belong to a specific Entity. There are also "non send resources", which can only be accessed on the main thread."

I wouldn't call Local a resource. It isn't stored the same way / World doesn't store them. Therefore we don't need to distinguish between "global" and "local" resources. Resources are "global" by definition.

Reply:
Interesting. They tend to behave like resources from the end-user's perspective in other ways, but I see your point. The main problems from a pedagogical perspective are a) "local" is an adjective, not a noun so it begs the "local *what*" question b) where should they be discussed from a teaching perspective.

Would you just call them "local unstructured data storages" or something? Do we start describing the ECS as having 3 places it can store data?

Reply:
@alice-i-cecile maybe we can call it "system local persistent data". It's persistent in the sense that it lasts across system calls but I hope it doesn't get confused with the more general "persistent storage" concept, opposed to "volatile storage".

Reply:
Right 🤔 I think I like that.

Reply:
Or "system local state". This time it can be confused with states though 😬

Reply:
I think "system local persistent data" is fine if it's followed by the explanation that it's only persistent between frames

Reply:
Yeah I think its enough to say that Local is "a persistent value stored locally in the system".